Assets/Scripts/Modules/Stats/IStatModule.cs

```diff
@@ -5,4 +5,5 @@
 
 public interface IStatModule {
 
+	void AssignTitle ();
```

Comment:
The Titles stat changes to the player so it should take an IUnit I imagine, but not IMapTile